_config.yml

```diff
@@ -88,6 +88,13 @@ avatarurl: "https://www.gravatar.com/avatar/6c0377abcf4da91cdd35dea4554b2a4c" #
 # service_worker: false # Will turn off the service worker if set to false
 css_inline: true # Will insert all styles into a single <style> block in the <head> element and remove the style <link> reference
 
+# Include additional javascript files in header and body, scripts must be in a relative path to the site
+# include_before_start_js:
+#  - /assets/scripts/my-awesome-header.js
+# 
+# include_before_end_js:
+#  - /assets/scripts/my-awesome-footer.js
```

Comment:
I think it would be more more versatile to other developers if we didn't check the config for listed js files. If the include were just empty html files a developer could put anything in there; e.g. CSS, HTML, embed code

Reply:
Not sure if I understand this. You mean simply including two empty html files in the layout and letting developers to fill it in with what ever they want? So in this case the configuration parameters are redundant. right?